src/test/java/edu/harvard/iq/dataverse/externaltools/ExternalToolServiceBeanTest.java

```diff
@@ -501,4 +501,47 @@ public void testParseAddToolWithLegacyType() {
         assertNull(externalTool.getContentType());
     }
 
+    @Test
+    public void testParseAddDatasetToolAllowedApiCalls() {
```

Comment:
Why not let the Exception bubble? It's common practice and just fine use `throws` with test cases instead of try/catch,

Reply:
Just following the local custom - many of the calls in this class do try/catch.